### ISSUE TITLE
[Issue]: MD.emu Shortcuts broken

### ISSUE THREAD

**Issue by LeRoy3rd** [Rakashazi/emu-ex-plus-alpha#319]
### Describe the bug

I own several of the .emu emulators (NES, neo, 2600, and MD). Recently the MD (Genesis) shortcuts won't launch the games. Each shortcut launches the app, instead. I thought maybe it was a storage location issue (as there seems to be a problem storing saved game files on the SD card), but changing that didn't help. It should be noted that I was having the same issue with NEO, but changing the save location fixed it in this case. I tried recreating shortcuts as well, to no avail. Little help? 

### Steps to reproduce

Click on a shortcut to a specific game in MD.emu

### Expected behavior

I'd expect it to load the linked game (it only launched MD.emu)

**Comment by Rakashazi** [comment]
What version of Android are you using and does this only affect shortcuts to games on the SD card, or internal storage too?

**Comment by LeRoy3rd** [comment]
Thanks for your reply! I'm on Android 8, and all roms are on my SD card. I've discovered another wrinkle that may help trace this down. I refuse to pirate games, I own the systems and a retrode to extract the data from my own games. HOWEVER, collections like the NeoGeo games from GoG.com, that come with the ROMs, or the Sega Collection on Steam (where you can extract the ROMs), I purchase these, and use those... In this instance, I've found that, while all games run, the shortcuts only fail on the games that were extracted from the Genesis collection. The games I extracted from my own carts still work fine when launching from the shortcuts.

It's also useful to note that I've extracted games from the Atari collection (2600), Disney afternoon (NES) collection, and megaman collection (NES), and those shortcuts still work. It's only the Genesis/MD games where the shortcut fails.

**Comment by Rakashazi** [comment]
Thanks for the additional info, I tested it out in an Android 8 VM but couldn't reproduce the issue so it may be something file path related. Give the new nightly build a try when it finishes compiling and see if it prints a red error popup at the bottom of the screen when loading one of the non-working shortcuts or if it still behaves like before.

**Comment by LeRoy3rd** [comment]
When the nightly update didn't resolve the issue, I did some more poking (which, admittedly, I should have done earlier)... I found if I removed the shortcuts, loaded the game, then recreated the shortcuts, the newly created shortcuts work... So, now I'm in the process of removing, re-adding, and coupling box art with each of the affected titles. I'm not sure what caused the issue, but, as I've found a workable solution, I'll close this ticket. Thanks for your willingness to attempt to address such a minor issue.

(as a feature request, I'd love the ability to batch-create shortcuts...)
